Products/ZenModel/ZenModelRM.py

```diff
@@ -86,6 +86,7 @@ def zmanage_editProperties(self, REQUEST=None, redirect=False, audit=True):
         return ZenModelBase.zmanage_editProperties(self, REQUEST, redirect, audit)
 
 
+    security.declareProtected('Manage DMD', 'zmanage_addProperty')
```

Comment:
Make sure we are using the constants and not magic strings when declaring permissions